airbyte-db/db-lib/build.gradle

```diff
@@ -58,53 +58,17 @@ dependencies {
 
 }
 
-task(newConfigsMigration, dependsOn: 'classes', type: JavaExec) {
-    mainClass = 'io.airbyte.db.instance.development.MigrationDevCenter'
-    classpath = files(configurations.migrations.files)
-    args 'configs', 'create'
-}
-
-task(runConfigsMigration, dependsOn: 'classes', type: JavaExec) {
-    mainClass = 'io.airbyte.db.instance.development.MigrationDevCenter'
-    classpath = files(configurations.migrations.files)
-    args 'configs', 'migrate'
-}
-
-task(dumpConfigsSchema, dependsOn: 'classes', type: JavaExec) {
-    mainClass = 'io.airbyte.db.instance.development.MigrationDevCenter'
-    classpath = files(configurations.migrations.files)
-    args 'configs', 'dump_schema'
-}
-
-task(newJobsMigration, dependsOn: 'classes', type: JavaExec) {
-    mainClass = 'io.airbyte.db.instance.development.MigrationDevCenter'
-    classpath = files(configurations.migrations.files)
-    args 'jobs', 'create'
-}
-
-task(runJobsMigration, dependsOn: 'classes', type: JavaExec) {
-    mainClass = 'io.airbyte.db.instance.development.MigrationDevCenter'
-    classpath = files(configurations.migrations.files)
-    args 'jobs', 'migrate'
-}
-
-task(dumpJobsSchema, dependsOn: 'classes', type: JavaExec) {
-    mainClass = 'io.airbyte.db.instance.development.MigrationDevCenter'
-    classpath = files(configurations.migrations.files)
-    args 'jobs', 'dump_schema'
-}
-
-task copyInitSql(type: Copy) {
```

Comment:
I think this + the `tasks.named("buildDockerImage")` block are still needed. Or at least, `:airbyte-db:db-lib:buildDockerImage` is failing because `/actions-runner/_work/airbyte/airbyte/airbyte-db/db-lib/build/docker` doesn't exist

Reply:
thanks @edgao . do you have any idea what this docker container is used? i was having trouble finding usages.

Reply:
oh nvm. i think it is exclusively used in the platform. so probably not needed here.

Reply:
ah nice, that makes sense. had thought this goes into one of the connectors base images but that wouldn't make sense (and is verifiably false anyway)

I think you also need to delete this line https://github.com/airbytehq/airbyte/blob/master/airbyte-db/db-lib/gradle.properties#L1 to remove the buildDockerImage task as well

Reply:
thanks!